features/src/main/feature/camel-features.xml

```diff
@@ -1128,10 +1128,12 @@
     <feature name='camel-http' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <feature version='[5,6)'>http-client</feature>
+        <feature version="[6,7)">jakarta-servlet</feature>
```

Comment:
Just a note: with Karaf 4.5.x (with Pax Web 10), we will support servlet 6 so this bundle won't be needed anymore.